core/src/components/menu-toggle/usage/react.md

```diff
@@ -0,0 +1,26 @@
+```tsx
+import React from 'react';
+import { IonMenu, IonHeader, IonToolbar, IonTitle, IonContent, IonList, IonItem, IonMenuToggle, IonButton } from '@ionic/react';
+
+export const MenuExample: React.FC = () => (
+  <>
+    <IonMenu side="start" menuId="first">
```

Comment:
Might be good to add a `contentId` here like the other examples, so this works out of the box.